projects/step-frontend/src/lib/modules/timeseries/performance-view/filter-bar/filter-bar.component.scss

```diff
@@ -83,12 +79,15 @@ step-ts-filter-bar {
     padding: 0 8px;
   }
 
+  /**
+  These got override by the theme scss file
+   */
   .apply-btn {
-    color: #4b97de;
-    font-size: 12px;
-    line-height: 28px;
-    margin-right: 12px;
-    font-weight: bold;
+    color: #4b97de !important;
```

Comment:
cant u simply rename the class? I cant see it being used elsewhere else

Reply:
Could you in stead use a more concrete selector like "div>button.apply-btn"?

Also, we should use a variable for the colour.

Reply:
Yes, will do